packages/dnd-timeline/src/hooks/useItem.tsx

```diff
@@ -34,6 +35,15 @@ const getDragDirection = (
 	return null;
 };
 
+const defaultCalculateNewSpan: CalculateNewSpan = ({
+	span,
+	direction,
+	deltaX,
```

Comment:
`deltaX` is in pixels, and `Span`'s `start` and `end` values should be transformed values, relative to the current range.  
  
Make it so the function receives translated values. You can name it `delta`, just make sure it receives translated values.  You can do so by calling the `getValueFromScreenX` and calculating the delta.